packages/compass-aggregations/src/modules/explain.ts

```diff
@@ -216,4 +220,21 @@ export const explainAggregation = (): ThunkAction<
   }
 };
 
+const _getExplainVerbosity = (
```

Comment:
NIT: Prefixing module functions that are not exported with `_` shouldn't be necessary. Are other functions in this file / package following this convention?

Normally we prefix private class methods with `_` to signal that a method is private without TS so it is clear that we don't need to call that when we use the class outside.